cylc/flow/resources.py

```diff
@@ -72,3 +84,20 @@ def get_resources(target_dir, resources=None):
         res = pr.resource_string('cylc.flow', resource)
         with open(path, 'wb') as h:
             h.write(res)
+
+
+def extract_tutorials():
+    """Extract tutorial workflows to Cylc Source Directory."""
+    src = Path(cylc.flow.__file__).parent / 'etc/tutorial'
+    dest = glbl_cfg().get(['install', 'source dirs'])[0]
```

Comment:
What if no source dirs are configured?

Reply:
Doesn't it have a default value in the config of `$HOME/cylc-src`?